modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,425 @@
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_POST_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters
+----
+
+== Enable PSC on an existing BYOVPC cluster
+
+. Update the Redpanda Cloud Agent IAM role.
++
+To allow the agent to create and manage the service attachment, add the following permissions to its IAM role:
++
+[,bash]
+----
+compute.forwardingRules.use
+compute.regionOperations.get
+compute.serviceAttachments.create
+compute.serviceAttachments.delete
+compute.serviceAttachments.get
+compute.serviceAttachments.list
+compute.serviceAttachments.update
+compute.subnetworks.use
+----
+
+. In the Redpanda Cloud UI, go to the cluster overview and copy the cluster ID from the **Details** section.
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to include the newly-created PSC NAT subnet and enable PSC.
++
+Set the following environment variables:
++
+--
+* `ACCEPT_LIST`: a JSON list specifying the projects from which incoming connections will be accepted. All other sources. For example, `[{"source": "consumer-project-ID-1"},{"source": "consumer-project-ID-2"}]`.
+* `PSC_NAT_SUBNET_NAME`: The name of the PSC NAT subnet. Use the fully-qualified name, for example `"projects/<PROJECT>/regions/<REGION>/subnetworks/<SUBNET-NAME>"`.
+--
++
+[,bash]
+----
+ACCEPT_LIST='[]'
+PSC_NAT_SUBNET_NAME='<PSC NAT subnet name>'
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+    "customer_managed_resources": {
+    "gcp": {
+        "psc_nat_subnet_name": "$PSC_NAT_SUBNET_NAME"
+    }
+    },
+    "private_link": {
+    "enabled": true,
+    "gcp": {
+        "consumer_accept_list": $ACCEPT_LIST
+    }
+    }
+}
+EOF`
+curl -v -X PATCH \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" ${PUBLIC_API_ENDPOINT}/v1beta1/clusters/$CLUSTER_ID
+----
++
+Wait for the cluster to apply the new configuration (around 15 minutes). The PSC service attachment is available when the cluster update is complete. You can monitor the service attachment creation by running the following `gcloud` command:
++
+[,bash]
+----
+gcloud compute service-attachments list --project '<service-project-ID>'
+----
+
+== Optional: Deploy consumer resources
+
+=== Deploy PSC consumer-side resources
+
+These resources will set up a connection to the service attachment created in the previous step.
+
+Make sure to provide your own values for the following:
+
+- `service_attachment_uri`: The URI to the service attachment. The format must be: `projects/<Redpanda-cluster-project-ID>/regions/<Redpanda-cluster-region>/serviceAttachments/<service-attachment-name>`.
+- `consumer_project_id`: The ID of the GCP project where the consumer (client) resources will be created. Should match one of the projects included in the `consumer_accept_list` field from previous steps.
```

Comment:
Between "consumer" vs "client" is one term preferred? What would be more clear to our users?

Reply:
I'd go with "consumer". It's the terminology used in the GCP PSC docs.